### PR TITLE
feat(integrations): Add Grafana and Insight Finder to the integrations directory

### PR DESCRIPTION
**Pull request by manuzope**
## Integration Directory view
Screen Shot 2021-11-29 at 3 28 53 PM
Screen Shot 2021-11-29 at 3 28 43 PM

## Integration detail view
Screen Shot 2021-11-29 at 3 29 13 PM
Screen Shot 2021-11-29 at 3 29 07 PM

## TODO:
- [x] Replace placeholder SVGs with Sentry approved SVGs
- [x] Fix the tests

### CODE REVIEW

**Comment by manuzope** [comment]
@jonesphillip How's the wording look? Specifically, around `features[].description`.

**Comment by jonesphillip** [comment]
> @jonesphillip How's the wording look? Specifically, around `features[].description`.

looks good to me

**Comment by jonesphillip** [comment]
@manuzope can we set the POPULARITY_WEIGHT to 10 for Grafana?

File: static/app/views/organizationIntegrations/constants.tsx [hunk 1]
```diff
@@ -234,6 +236,46 @@ export const documentIntegrationList: DocumentIntegration[] = [
...
+      },
+    ],
+  },
+  {
+    slug: 'insight_finder',
+    name: 'Insight Finder',
+    author: 'Insight Finder',
+    docUrl: 'https://insightfinder.com/insightfinder-sentry-integration/',
+    description:
+      'InsightFinder ingests the errors that Sentry detects through its standard APIs and analyze it using our patented, unsupervised, neural network algorithms. InsightFinder prioritizes those errors and provides context so anomalous events can be resolved before business is impacted.',
```
Comment (ceorourke) [inline static/app/views/organizationIntegrations/constants.tsx:265]: I think "analyze" should be "analyzes" here

Comment (ceorourke) [reply static/app/views/organizationIntegrations/constants.tsx:265]: and something like "analyzes them" because it's referring to "errors"

**Review by mgaeta — APPROVED** [review]

**Review by leeandher — APPROVED** [review]

**Review by ceorourke — APPROVED** [review]

**Review by NisanthanNanthakumar — APPROVED** [review]

**Review by AniketDas-Tekky — APPROVED** [review]

**Pull request merged**